string image_topic
string file_name
---
bool success
string message
